frontend/src/components/common/pagination/pagination.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'common/types/types';
+import { useEffect, useState } from 'hooks/hooks';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  totalCount: number;
+  pageSize?: number;
+  currentPage: number;
+  onPageChange: (newPage: number) => void;
+};
+
+const Pagination: FC<Props> = ({
+  totalCount,
+  pageSize = 5,
```

Comment:
Do not use magic numbers - Farid in his PR related to pagination has defined pagination constants - they may be declared in shared folder and used both for BE and FE